### PR TITLE
Add caching for static assets

### PR DESCRIPTION
**Pull request by gideonthomas**
cc @mmmavis to make sure you're ok with this.

Added this per https://github.com/MozillaFoundation/mofo-devops/issues/623#issuecomment-430667951

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -28,7 +28,10 @@ var limiter = RateLimit({
 });
 
 app.use(compression());
-app.use(express.static(path.resolve(__dirname, `public`)));
+app.use(express.static(path.resolve(__dirname, `public`), {
+  maxAge: '1m', // expire cache in 1 month
+  immutable: true
```
Comment (cadecairos) [inline server.js]: Are you sure we should add this directive? It seems like clients that support it wouldn't even bother to check for updates on a force refresh?

Comment (gideonthomas) [reply server.js]: do you think static assets would ever need to be updated though? We would just change the name of the asset wouldn't we?

Comment (cadecairos) [reply server.js]: are we serving any compiled JS or CSS bundles?

Comment (cadecairos) [reply server.js]: (and yeah, if we're prefixing the filename with a hash then we don't have a problem, but that assumes we're already doing that, so... are we?)

Comment (gideonthomas) [reply server.js]: good catch: https://github.com/mozilla/mozillafestival.org/blob/master/webpack.config.js#L21, I'll remove it

**Comment by mmmavis** [comment]
@cadecairos you okay with this PR? 👀

**Pull request merged**